src/components/widgets/RecentListWidget.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+	<Card :loading="loading">
+		<template #title>
+			{{getNameByKey('recent')}} {{getNameByKey(title)}}
```

Comment:
There may be a problem with some languages (eg. word order). I would use single key for translation.
```suggestion
			{{getNameByKey(title)}}
```